cyclonedx.go

```diff
@@ -1124,12 +1127,14 @@ type Note struct {
 }
 
 type OrganizationalContact struct {
-	Name  string `json:"name,omitempty" xml:"name,omitempty"`
-	Email string `json:"email,omitempty" xml:"email,omitempty"`
-	Phone string `json:"phone,omitempty" xml:"phone,omitempty"`
+	BOMRef string `json:"bom-ref,omitempty" xml:"bom-ref,attr,omitempty"`
```

Comment:
Should `BOMRef` be emptied in `convert.go` for spec versions lower than `1.6`?

Reply:
The `BOMRef` was added to `OrganizationalContact` in the [1.5 spec](https://github.com/CycloneDX/specification/blob/master/schema/bom-1.5.schema.json#L363-L367) but are currently missing. You're right, they will need converting for SBOMs `< 1.5`.

It's probably easier to pull this change into its own PR and update the test SBOMs in this PR by removing the `bom-ref`s.

Reply:
I've moved the `BOMRef` changes to [this PR](https://github.com/CycloneDX/cyclonedx-go/pull/179).